### PR TITLE
Mapped Buttons

### LINKED ISSUE #81: Update controller mappings per new diagram

### PR DESCRIPTION
**Pull request by AlenaNguyen**
I had a little trouble with the game piece override; does that code seem like it should work?

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Transfer.java [hunk 1]
```diff
@@ -47,7 +48,27 @@ public void setFeederNeutralOutput() {
   }
 
   public boolean getGamePieceCollected() {
-    return (constTransfer.NOTE_SENSOR_INVERT) ? !noteSensor.get() : noteSensor.get();
+    // return (constTransfer.NOTE_SENSOR_INVERT) ? !noteSensor.get() :
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/subsystems/Transfer.java]: You can make this code a bit cleaner by moving the value of the note sensor into it's own line:

boolean noteSensorValue = (constTransfer.NOTE_SENSOR_INVERT) ? !noteSensor.get() : noteSensor.get();
return (noteSensorValue  || hasGamePiece);

Comment (alicekuznetsov) [reply src/main/java/frc/robot/subsystems/Transfer.java]: I think the implementation of should work though!! We should test it before we merge it in

**Review by alicekuznetsov — DISMISSED** [review]
Can you also try to resolve #73 in this pr? I think they're similar enough where we can do them both now, and its basically the only other thing we need to program apart from the climb/trap

**Comment by alicekuznetsov** [comment]
I'll also see if i can find time to make a proper diagram and throw it in our readme :p 
If i do, I'll throw it in this branch. If i don't, just merge this in without it

**Comment by TaylerUva** [comment]
Btw this is one that we can test really well in sim

**Comment by TaylerUva** [comment]
- [ ] Can't go from prep_vision to other prep_*
- [x] DPad west is incorrect. It should be shuffle. Currently wing

FYI Note: can't do shoot in sim anymore (not issue for now)

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**